.eslintrc

```diff
@@ -0,0 +1,158 @@
+rules:
+  indent:
+    - 2
+    - 2
+  quotes:
+    - 2
+    - single
+  linebreak-style:
+    - 2
+    - unix
+  semi:
+    - 2
+    - never
+  array-bracket-spacing: 
+    - 2
+    - "never"
+  block-scoped-var: 2
+  brace-style: 
+    - 2
+    - "stroustrup"
+    - 
+      allowSingleLine: true
+  comma-dangle: 
+    - 2
+    - "never"
```

Comment:
I think linting solves a lot of those problems, and most often a missing comma is a syntax error that gets flagged by the parser right away. Comma first can actually look pretty good to me, but my biggest issue is that I haven't been able to get js-mode in emacs to play nice with indenting it properly. I also think that having each `var` definition on its own line is more readable than bunching them all into one statement.